vendor/golang.org/x/tools/internal/pkgbits/decoder.go

```diff
@@ -73,6 +76,7 @@ func (pr *PkgDecoder) SyncMarkers() bool { return pr.sync }
 func NewPkgDecoder(pkgPath, input string) PkgDecoder {
 	pr := PkgDecoder{
 		pkgPath: pkgPath,
+		//aliases: aliases.Enabled(),
```

Comment:
This line of diff is vaguely concerning to me. Is this not supposed to be commented out? Is this supposed to be followed-up with a TODO? I've [asked upstream](https://go-review.googlesource.com/c/tools/+/580455/4#message-4e5c55452a2afab8df79aed87236f41348b9613e).